### ISSUE TITLE
[BUG] v0.15.0 does not create climate entities anymore

### ISSUE THREAD

**Issue by wburgers** [plugwise/plugwise-beta#176]
**Describe the bug**
I am using an Anna in my living room, with Adam near my central heater and Tom radiator valves in each room.
After upgrading from 0.14.7 to v0.15.0, there is no climate entity in HA for the radiator valve for any of the rooms I only have a Tom.

**Evidence**
There is no error in the logs whatsoever. No debug logging either, even after setting the config to:
```
logger:
  default: warn
  logs:
    custom_components.plugwise-beta: debug
```

**Expected behavior**
I expect to have a climate entity for each room, such that I can set the right scene for each room.
I believe if you set the zones in Adam correctly you can detect Anna in the living room and not create a climate entity for the Tom devices in the living room (as it was in 0.14.x), but for the other rooms I think a climate entity is needed.

**Screenshots**
Before removing the entity, it is restored from the database, but does show the red warning on the right:
![image](https://user-images.githubusercontent.com/2129305/122407705-1d58f780-cf82-11eb-9830-5c76a4dbb6dc.png)
After removing the entity, it is not visible anymore for the device:
![image](https://user-images.githubusercontent.com/2129305/122409036-1e3e5900-cf83-11eb-8a7b-4014fda2092d.png)

And of course lovelace does not show the thermostat card anymore.

**Home Assistant (please complete the following information):**
 - Version: 2021.6.5
 - Deployment: Container: raspberrypi2-homeassistant:stable
 - Installed plugwise version: plugwise-beta 0.15.0
 - Installed through: HACS

**Smile (please complete the following information):**
 - Sort of Smile: HA connects to Adam. Anna is discovered through Adam
 - Firmware: 3.2.8, 
 
**Additional context**
-

**Comment by bouwew** [comment]
@wburgers I understand your bug I think, I will check the code for bugs in relation to this bug-report.

**Comment by bouwew** [comment]
@wburgers can you please test plugwise-beta v0.15.1b0?

**Comment by bouwew** [comment]
That's good news!
I've made so many changes to both the plugwise-beta v0.15.0 as the python-plugwise v0.10.0 code, I missed one small thing, sorry!

Thanks for testing, I'll release the new version.

**Comment by wburgers** [comment]
No worries, you don't have to apologise!
I appreciate your hard work on this component and I use it with much pleasure.
I always tell every Tado user that the Plugwise integration is much better and provides much more stuff 😁.

Thanks again!

**Comment by bouwew** [comment]
Closing this issue, solved in #177
